.drone.yaml

```diff
@@ -0,0 +1,80 @@
+ ---
+kind: pipeline
+name: linux-amd64
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: ci
+  image: briandowns/rancher-build-base:v0.1.1
```

Comment:
We shouldn't be using this image anymore we can probably use what this image has been replaced with, `rancher/hardened-build-base` or something like the golang image.